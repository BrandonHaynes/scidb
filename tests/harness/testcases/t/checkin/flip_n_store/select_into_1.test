--setup
--start-query-logging
create array A <a:int64> [x=0:2,3,0]
create array B <x:int64> [a=0:15,3,0]
store(build(A,x+10),A)

--test

--aql select * into B from A

--aql select a,x from B

--aql select * into B from A as Az

--aql select a,x from B

--aql select count(B)

--aql select * into B from A where a=11

--aql select a,x from B

--aql select * into B from A where a<>11

--aql select a,x from B

--aql select * into B from A where x>0

--aql select a,x from B

--aql select * into B from A where x<=1

--aql select a,x from B

--aql select a into B from A

--aql select a,x from B

--aql select x into B from A

--aql select a,x from B

--aql select a,x into B from A

--aql select a,x from B

--aql select x,a into B from A

--aql select a,x from B

--cleanup
remove(A)
remove(B)
--stop-query-logging
