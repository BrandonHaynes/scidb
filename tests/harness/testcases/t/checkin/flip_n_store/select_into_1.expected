SCIDB QUERY : <create array A <a:int64> [x=0:2,3,0]>
Query was executed successfully

SCIDB QUERY : <create array B <x:int64> [a=0:15,3,0]>
Query was executed successfully

SCIDB QUERY : <store(build(A,x+10),A)>
{x} a
{0} 10
{1} 11
{2} 12

SCIDB QUERY : <select * into B from A>
{a} x
{0} 10
{1} 11
{2} 12

SCIDB QUERY : <select a,x from B>
{a} expr,x
{0} 0,10
{1} 1,11
{2} 2,12

SCIDB QUERY : <select * into B from A as Az>
{a} x
{0} 10
{1} 11
{2} 12

SCIDB QUERY : <select a,x from B>
{a} expr,x
{0} 0,10
{1} 1,11
{2} 2,12

SCIDB QUERY : <select count(B)>
{i} count
{0} 3

SCIDB QUERY : <select * into B from A where a=11>
{a} x
{1} 11

SCIDB QUERY : <select a,x from B>
{a} expr,x
{1} 1,11

SCIDB QUERY : <select * into B from A where a<>11>
{a} x
{0} 10
{2} 12

SCIDB QUERY : <select a,x from B>
{a} expr,x
{0} 0,10
{2} 2,12

SCIDB QUERY : <select * into B from A where x>0>
{a} x
{1} 11
{2} 12

SCIDB QUERY : <select a,x from B>
{a} expr,x
{1} 1,11
{2} 2,12

SCIDB QUERY : <select * into B from A where x<=1>
{a} x
{0} 10
{1} 11

SCIDB QUERY : <select a,x from B>
{a} expr,x
{0} 0,10
{1} 1,11

SCIDB QUERY : <select a into B from A>
{a} x
{0} 10
{1} 11
{2} 12

SCIDB QUERY : <select a,x from B>
{a} expr,x
{0} 0,10
{1} 1,11
{2} 2,12

SCIDB QUERY : <select x into B from A>
{a} x
{0} 0
{1} 1
{2} 2

SCIDB QUERY : <select a,x from B>
{a} expr,x
{0} 0,0
{1} 1,1
{2} 2,2

SCIDB QUERY : <select a,x into B from A>
{a} x
{10} 0
{11} 1
{12} 2

SCIDB QUERY : <select a,x from B>
{a} expr,x
{10} 10,0
{11} 11,1
{12} 12,2

SCIDB QUERY : <select x,a into B from A>
{a} x
{10} 0
{11} 1
{12} 2

SCIDB QUERY : <select a,x from B>
{a} expr,x
{10} 10,0
{11} 11,1
{12} 12,2

SCIDB QUERY : <remove(A)>
Query was executed successfully

SCIDB QUERY : <remove(B)>
Query was executed successfully

